### PR TITLE
fix(opencensus): add core dependency

### LINKED ISSUE #7239: Opencensus Metric collection example failing with ClassNotFoundException
Quickstart example that is given in the opencensus repo failing with below exception.

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/api/client/json/jackson2/JacksonFactory
	at io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration.<clinit>(StackdriverStatsConfiguration.java:50)
	at com.example.opencensus.Quickstart.main(Quickstart.java:26)
Caused by: java.lang.ClassNotFoundException: com.google.api.client.json.jackson2.JacksonFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 2 more
```

I have just cloned the repo into my machine to ran the example. Did not make changes to either code or pom.xml, still I a getting the aboe exception.

Am I missing something here?

### PR DESCRIPTION
**Pull request by kweinmeister**
Fixes #7239

### CODE REVIEW

File: opencensus/pom.xml [hunk 1]
```diff
@@ -39,6 +39,20 @@
...
   </properties>
 
+ <!--  Using libraries-bom to manage versions.
+  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>25.0.0</version>
```
Comment (kolea2) [inline opencensus/pom.xml:49]: Please update to the latest version: https://search.maven.org/artifact/com.google.cloud/libraries-bom/26.1.3/pom

File: opencensus/pom.xml [hunk 2]
```diff
@@ -56,6 +70,10 @@
       <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
       <version>${opencensus.version}</version>
     </dependency>
+    <dependency>
+			<groupId>com.google.cloud</groupId>
```
Comment (kolea2) [inline opencensus/pom.xml:74]: please fix the indentation

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: opencensus/pom.xml (hunk 1 above)
Comment (kweinmeister) [reply opencensus/pom.xml]: Done!

File: opencensus/pom.xml (hunk 2 above)
Comment (kweinmeister) [reply opencensus/pom.xml]: It looks like it's a formatting issue here in the diff UI. I double-checked in the editor and the formatting is correct. Also, if you look at the "Files changed" tab, it shows up properly. Please LMK if there's something else I can do here.

**Review by kolea2 — APPROVED** [review]

File: opencensus/pom.xml (hunk 2 above)
Comment (kolea2) [reply opencensus/pom.xml]: sg, thanks!

**Pull request merged**